data/org.mate.panel.menubar.gschema.xml.in

```diff
@@ -50,5 +50,10 @@
       <summary>Threshold of menu items before submenu is created</summary>
       <description>Maximum number of menu items (i.e. bookmarks) that are displayed without being put in a submenu.</description>
     </key>
+    <key name="max-recent-items" type="i">
+      <default>10</default>
```

Comment:
It is better to add the setting range of the value

Reply:
Roger that.  Fix applied.

Note that I still don't specify a maximum (I only specify a minimum of -1) since the property I modify has no clear maximum value:  https://developer.gnome.org/gtk3/3.24/GtkRecentChooser.html#GtkRecentChooser--limit